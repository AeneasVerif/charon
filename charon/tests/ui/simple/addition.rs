fn main() {
    let _ = 1 + 2;
}
