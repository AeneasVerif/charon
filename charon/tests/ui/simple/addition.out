# Final LLBC before serialization:

fn UNIT_METADATA()
{
    let @0: (); // return

    @0 := ()
    return
}

const UNIT_METADATA: () = @Fun0()

// Full name: test_crate::main
fn main()
{
    let @0: (); // return
    let @1: i32; // anonymous local
    let @2: i32; // anonymous local

    storage_live(@2)
    @0 := ()
    storage_live(@1)
    @2 := const (1 : i32) panic.+ const (2 : i32)
    @1 := move (@2)
    storage_dead(@1)
    @0 := ()
    return
}



